tests/unit/array/conftest.py

```diff
@@ -13,7 +13,7 @@ def start_storage():
         f"docker-compose -f {compose_yml} --project-directory . up  --build -d "
         f"--remove-orphans"
     )
-    time.sleep(5)
+    time.sleep(20)
```

Comment:
I'm worried 20s is too large, and our CI workflow is too slow.

Reply:
it takes much more time for elastic to be usable than other docker images, I've tried with 10 and 15 and didn't worked.

Reply:
I see